OmniKit/MessageBlocks/ConfigureAlertsCommand.swift

```diff
@@ -108,21 +122,24 @@ public struct ConfigureAlertsCommand : MessageBlock {
             self.alertType = alertType
             
             self.audible = encodedData[0] & 0b1000 != 0
-
+            
             self.autoOffModifier = encodedData[0] & 0b10 != 0
-
+            
             self.duration = TimeInterval(minutes: Double((Int(encodedData[0] & 0b1) << 8) + Int(encodedData[1])))
-
+            
             let yyyy = (Int(encodedData[2]) << 8) + (Int(encodedData[3])) & 0x3fff
-
+            
             if encodedData[0] & 0b100 != 0 {
                 let volume = Double(yyyy * 2) * podPulseSize
                 self.expirationType = .reservoir(volume: volume)
             } else {
                 self.expirationType = .time(TimeInterval(minutes: Double(yyyy)))
             }
             
-            self.beepType = (UInt16(encodedData[4]) << 8) + UInt16(encodedData[5])
+            self.beepType = BeepType(rawValue: encodedData[4])!
```

Comment:
This should throw an exception if the rawValue isn't valid

Reply:
Do you have an example how I can assert this? Would be good to write a test to see if it fails on values higher than 8.

Reply:
```